code/game/objects/items/devices/scanners.dm

```diff
@@ -248,6 +248,14 @@ proc/healthanalyze(mob/living/M as mob, mob/living/user as mob, var/mode = 0)
 	melt_temperature = MELTPOINT_PLASTIC
 	origin_tech = "magnets=1;engineering=1"
 
+	var/list/human_standards = list("oxygen_min" = 16,

```

Comment:
this is one of those non-oop legacy uses of the IDs that need to be fixed at some point